src/components/campaignLandingPage/query.ts

```diff
@@ -118,18 +122,13 @@ export const formatter: QueryFormatter<
         queries.formatData('media--video', entity.field_media),
       button:
         entity.field_clp_video_panel_more_video &&
-        queries.formatData(
-          'paragraph--button',
-          entity.field_clp_video_panel_more_video
-        ),
+        buttonFormatter(entity.field_clp_video_panel_more_video),
```

Comment:
Is there a special reason for this change like something wasn't working correctly? Or do you just prefer the explicit imports and direct calls?

Reply:
I personally like it to be more explicit myself. Like, if we're already calling it with the `'paragraph--button'` magic string, we can cut out the `next-drupal` layer entirely. This, in fact, will even make upgrading to the next major version easier later down the road. Or if we decide to move away from it entirely, it's at least not _harder_ to make that migration. 😇